src/common/CountOverlay.js

```diff
@@ -27,7 +27,7 @@ export default class CountOverlay extends PureComponent<Props> {
       <View>
         <ComponentWithOverlay
           style={styles.button}
-          overlaySize={15}
+          overlaySize={18}
```

Comment:
Aha, I have managed to track down the internal `@react-navigation/bottom-tabs` code that sets this size in the bottom-tab badge! 🙂

18 is very close! It looks like what we want to match is 25 * 0.75, which is 18.75: https://github.com/react-navigation/react-navigation/blob/%40react-navigation/bottom-tabs%405.11.15/packages/bottom-tabs/src/views/TabBarIcon.tsx#L69

So let's use 18.75, with a code comment linking to that line as the thing we want to match.

Reply:
Sure @chrisbobbe!, even I also, tracked the `@react-navigation/bottom-tabs` code and found that they were using size somewhere near 18. 
Sure, If our requirement is change the size to 18.75, we will do this.